uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/app/AzureSampleApp.java

```diff
@@ -128,6 +128,12 @@ public void signOutFromSingleAccountFragment() {
      */
     public void confirmSignedIn(@NonNull final String username) {
         Logger.i(TAG, "Confirming account with supplied username is signed in..");
+        try {
+            Thread.sleep(TimeUnit.SECONDS.toMillis(5));
```

Comment:
Looks like a huge sleep. I'm wondering what changed here that caused us to from zero sleep to 5 seconds of sleep here?

Reply:
Some test cases would fail because the signed-in username was not updated in time. We could also solve this by introducing a RetryOnFailure, but then we might rerun the whole test instead of doing the wait here.

Reply:
We could use waitForExists() instead, actually. Might do a later change for this.